low-level/sensors/impl/centos_7/disk_monitor.py

```diff
@@ -1082,8 +1109,9 @@ def _update_drive_faults(self):
                 except KeyError:
                     is_drive_faulty = True
 
-            if not self._existing_drive[object_path] and is_drive_faulty:
-                self._existing_drive[object_path] = True
+            if not self._existing_drive[object_path]['is_faulty'] and \
+                is_drive_faulty:

```

Comment:
rename is_drive_faulty -> fault_detected to avoid confusion and relate it to current drive fault state

Reply:
addressed. 👍